api/redisfailover/v1/types.go

```diff
@@ -175,6 +175,7 @@ type BootstrapSettings struct {
 	Host           string `json:"host,omitempty"`
 	Port           string `json:"port,omitempty"`
 	AllowSentinels bool   `json:"allowSentinels,omitempty"`
+	Enabled        bool   `json:"enabled,omitempty"`
```

Comment:
This changes the semantics of existing CRs which omit the `enabled:` value. Using this version of the operator would - semantically - disable bootstrapping because `enabled: nil` would be interpreted as `enabled: false`.

It feels a bit weird, but had we considered adding a [default value](https://kubebuilder.io/reference/markers/crd-validation) for backward compatibility when the CR omits the `Enabled` value?

```suggestion
        // +kubebuilder:default:true
	Enabled        bool   `json:"enabled,omitempty"`
```